data/importToDb.js

```diff
@@ -11,28 +11,38 @@ DatabaseManager.establishConnection().then(() => {
         DatabaseManager.getDatabase().beginTransaction();
 
         // FIXME: The driver does not support prepared statements, yet. Rewrite once it is supported.
-        const rawData = require('./rawData-frankfurt.geo');
+        const rawData = require('./rawData-bw.geo');
```

Comment:
I would suggest making this file query dynamic and not static with a particular suffix.

Reply:
Good idea, however this is only an optimization and not really a complete refactor. I would work on this after this has been merged. Shall I revert this line change for now?

Reply:
I will approve this PR now, as this is just an improvement, but I do not want to stop all the PR right now.